app/models/host_alias.rb

```diff
@@ -0,0 +1,13 @@
+class HostAlias < ActiveRecord::Base
+  belongs_to :domain
```

Comment:
i mean it the other way around, only one server can have an alias for api.foobar.com